roles.md

```diff
@@ -2,8 +2,125 @@
 layout: default
 ---
 
-### Roles
+# Roles
 
-## The Good Docs Project Steering Committee
+Here we define roles within The Good Docs Project, along with the eligibility, expectations, and responsibilities associated with these roles.
 
-*   Information about the Steering Committee can be found [here](https://github.com/thegooddocsproject/governance/blob/master/ProjectSteeringCommittee.md).
+**Version:** 2.0
+
+**Last updated:** August 2020
+
+## Community
+
+Everyone contributing to, or using artifacts from, The Good Docs Project is considered part of our community. We encourage all community members to actively participate in our activities. This includes:
+
+*   Participate in our forums and vote on motions.
+*   Raise and comment on bugs.
+*   Suggest fixes by raising pull requests.
+
+**Access:**
+
+*   Community members are granted _read_ access to our github repositories.
+
+**Expectations of community members:**
+
+*   Community members are expected to be respectful to each other, in line with our [Code of Conduct](https://github.com/thegooddocsproject/governance/blob/master/CodeOfConduct.md).
```

Comment:
Think it makes sense we create a set of onboarding materials that we expect them to complete.  have thoughts here.